api/scanner/media_encoding/executable_worker/magick_cli.go

```diff
@@ -0,0 +1,60 @@
+package executable_worker
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/photoview/photoview/api/utils"
+)
+
+type MagickCli struct {
+	path string
+}
+
+func newMagickCli() *MagickCli {
+	if utils.EnvDisableRawProcessing.GetBool() {
+		log.Printf("Executable worker disabled (%s=1): ImageMagick\n", utils.EnvDisableRawProcessing.GetName())
```

Comment:
Assuming that we log only the variable name. Don't we need to also include the boolean value as well?

Reply:
Updated.